Intersect.Editor/Forms/DockingElements/frmMapLayers.cs

```diff
@@ -1065,9 +1065,9 @@ private void InitLocalization()
             grpSlide.Text = Strings.Attributes.Slide;
             lblSlideDir.Text = Strings.Attributes.Direction;
             cmbSlideDir.Items.Clear();
-            for (var i = -1; i < 4; i++)
+            for (var i = Direction.None; i < (Direction)4; i++)
```

Comment:
Do Direction i = 0 like below

Do you really need to cast from 4 to Direction?

Since Direction is actually an int enum I would think the cast isn't needed

Reply:
some cases like these use -1 and others use 0 ; are you sure that we should change all these to 0 ?

Reply:
(?)

```csharp
            cmbSlideDir.Items.Clear();
            for (var i = 0; i < 4; i++)
            {
                cmbSlideDir.Items.Add(Strings.Direction.dir[(Direction)i]);
            }
```

Reply:
I think I replied in the PR contradicting myself. Just leave them as -1 if they were -1 before.

If you can easily figure out _why_ they are different (I am guessing they display "None" in the editor, but I am not sure why, it might be more clear looking at it from the perspective a user), please comment in each case that we start from -1.